main_test.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+)
+
+var a Job
+
+func TestMain(m *testing.M) {
+
+	a.Initialize("DB_USERNAME", "DB_PASSWORD", "rest_api_example")
+	ensureTableExists()
+	code := m.Run()
+	clearTable()
```

Comment:
use `setup()` or `teardown` for this to avoid repetition: http://www.inanzzz.com/index.php/post/2t08/using-setup-and-teardown-in-golang-unit-tests